bot/commands/Commands.py

```diff
@@ -37,13 +37,17 @@ def __init__(self):
 
             output = ""
             oceans = self.taskMgr.getOceanPopulations()
+            s = self.taskMgr.getSystemStatus()
             total = 0
 
             for i, k in sorted(oceans.items()):
                 output += "%s: %s\n" % (i, k)
                 total += k
 
-            output += BotLocalizer.OCEANS_TOTAL % total
+            if s.get('status', 0) == 3:
+                output = 'The Legend of Pirates Online is currently closed for an update. Ocean data is unavailable. Try again when the servers are back up!'
+            else:
+			    output += BotLocalizer.OCEANS_TOTAL % total
```

Comment:
May you please fix this line.  We require spaces, not tabs.  It's also indented too far.